src/resources/postcss/freemius.pcss

```diff
@@ -95,6 +95,10 @@
 				&:last-child {
 					margin-bottom: 0;
 				}
+
+				&:last-of-type {
```

Comment:
@paulmskim the above rule (last-child) doesn't apply because the Freemius logo in the upper corener is actually the last child - so we have extra margin here: http://p.tri.be/t8dow2